docs/imageset-config-ref.yaml

```diff
@@ -17,11 +17,10 @@ mirror:
     - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.8 # References entire catalog
       headsOnly: true # References latest version of each operator in catalog (true is the default value and can be omitted)
       packages:
-        - name: couchbase-operator # Planned, overrides catalog default, respects headsOnly setting
```

Comment:
Does this need to be changed back?

Reply:
I updated this just to give an example of a real package that existed in the catalog because I noticed providing invalid packages in the config ref was causing some confusion for users.